esmvaltool/diag_scripts/shared/plot/style.ncl

```diff
@@ -73,7 +73,7 @@ begin
   enter_msg(scriptname, funcname)
 
   ; Check for the available dictonary keys to be used for the annotations
-  dkeys = (/"dataset", "ensemble", "exp", "mip"/)
+  dkeys = (/"alias", "dataset", "ensemble", "exp", "mip"/)
```

Comment:
I think this would change the behavior of the function in a non-backward compatible way. E.g. instead of returning "ERA5" as dataset name (when called with option `annots`) `project_style` would return e.g. "native6" (as far as I understand the code).
I guess an option might be to implement a switch to allow the user to either use "alias" or "dataset". As default, "dataset" should be used to maintain backward compatibility.

Reply:
Sorry for the delay (I was without internet yesterday). I will have a look if I find a good solution by keeping backward compatibility in mind.

Reply:
@axel-lauer I made a new commit, which includes the option of defining the chosen annotations: ```annots: alias``` per namelist entry. If it is not there the default behaviour is reestablished. What do you think about this solution?

Reply:
Thanks for revising your contribution. I tested this and it works great and is now backward compatible. As an idea, the new option could also be added to the other perfmetrics diagnostics for improved consistency. This would probably be done best by adding the option to `main.ncl` which then passes it to the individual diagnostics.

The new option, however, needs to mentioned in the documentation of `cycle_zonal.ncl` and `zonal.ncl` (https://github.com/ESMValGroup/ESMValTool/blob/aliases_in_perfmetrics/doc/sphinx/source/recipes/recipe_perfmetrics.rst). After adding `annots` to the documentation, this is ready to be merged.

Reply:
Greta, thank you for testing it. I will add it to the documentation (just wanted to wait until the solution is reviewed). As I probably will work again with the perfmetrics diagnostic I can have a look how it can be added to the other perfmetrics diagnostics as well. I added it to cycle_zonal.ncl because it was similar to zonal.ncl, but with the others I am currently not sure how the annotations are applied. It will take some time

Reply:
Thanks @FranziskaWinterstein! I will approve this PR once the documentation has been updated, so hopefully in time for v2.8.

Reply:
Maybe one more thing, it would be nice if you could also revert the renaming of the loop index in `zonal.ncl` as this should already be fixed by #3098.